.github/workflows/osx_win_python_builds.yml

```diff
@@ -459,7 +459,8 @@ jobs:
         run: |
           python -m pip install dist/*cp313-win_amd64.whl
           python -m valhalla valhalla_service -h
-          python -m valhalla valhalla_build_tiles -h 
+          python -m valhalla valhalla_build_tiles -h
+          python -c "from valhalla.utils.graph_utils import GraphId; g = GraphId(); assert g.Is_Valid() == False"
```

Comment:
we don't test the python bindings on windows yet, so this is doing a bit more than only calling the executables